config/software/dep-selector-libgecode.rb

```diff
@@ -36,8 +38,11 @@
 
   # Ruby DevKit ships with BSD Tar
   env["PROG_TAR"] = "bsdtar" if windows?
+  env["ARFLAGS"] = "rv #{env["ARFLAGS"]}" if env["ARFLAGS"]
```

Comment:
Yeah...  oddities of make.  If you override it manually, you need to tack on the rv yourself.  Even more fun is the fact that the default changes based on version of make/platform you're on.  Because consistency....